src/shims/windows/foreign_items.rs

```diff
@@ -758,6 +758,22 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
                 this.write_null(dest)?;
             }
 
+            "_Unwind_RaiseException" => {
```

Comment:
@bjorn3 looks like this was added in the category of "Incomplete shims that we "stub out" just to get pre-main initialization code to work"; could you file a follow-up PR to move it to a more suitable location in this file?

Reply:
Also, the `tests/pass/panic/unwind_dwarf.rs` test should be run on windows-gnu targets to make sure we have test coverage.